CMakeLists.txt

```diff
@@ -40,11 +40,35 @@ target_link_libraries(KrillKounter PUBLIC ${CMAKE_DL_LIBS})
 # We link C++ files
 set_target_properties(KrillKounter PROPERTIES LINKER_LANGUAGE CXX)
 
+
+# Shared Library
+add_library(krillkounter SHARED ${sources})
```

Comment:
Does this mean the KrillKounter executable isn't linked against it's own library?